packages/select/index.js

```diff
@@ -0,0 +1,89 @@
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { when } from 'lit/directives/when.js';
+import { classNames } from '@chbphone55/classnames';
+import { kebabCaseAttributes, FabricElement } from '../utils';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+
+export class FabricSelect extends kebabCaseAttributes(FabricElement) {
+  static properties = {
+    // Whether the element should receive focus on render
+    autoFocus: { type: Boolean, reflect: true },
+
+    // Renders the field in an invalid state. Often paired with `hint` to provide feedback about the error
+    invalid: { type: Boolean, reflect: true },
+
+    // Whether to always show a hint
+    always: { type: Boolean, reflect: true },
+
+    // The content displayed as the help text
+    hint: { type: String, reflect: true },
+
+    // The element's unique identifier
+    id: { type: String, reflect: true },
+
+    // The content to disply as the label
+    label: { type: String, reflect: true },
+
+    // Whether to show optional text
+    optional: { type: Boolean, reflect: true },
+
+    _options: { state: true },
+  };
+
+  get #classes() {
+    return classNames('input mb-0', {
+      'input--is-invalid': this.invalid,
+    });
+  }
+
+  get #id() {
+    return this.id || `m${Math.random().toString(36).slice(2)}`;
```

Comment:
Not a bit deal, just a thought that this logic was also used in Attention component, and possibly in other use cases in the future, so maybe it makes sense to create a util that returns an id generated this way?